omod/src/main/java/org/openmrs/module/ohricore/extension/html/AdminList.java

```diff
@@ -13,39 +13,38 @@
 import java.util.Map;
 
 import org.openmrs.module.Extension;
-import org.openmrs.module.web.extension.AdministrationSectionExt;
 
 /**
  * This class defines the links that will appear on the administration page under the
  * "ohricore.title" heading. This extension is enabled by defining (uncommenting) it in the
  * config.xml file.
  */
-public class AdminList extends AdministrationSectionExt {
-	
-	/**
-	 * @see org.openmrs.module.web.extension.AdministrationSectionExt#getMediaType()
-	 */
-	public Extension.MEDIA_TYPE getMediaType() {
-		return Extension.MEDIA_TYPE.html;
-	}
-	
-	/**
-	 * @see org.openmrs.module.web.extension.AdministrationSectionExt#getTitle()
-	 */
-	public String getTitle() {
-		return "ohricore.title";
-	}
-	
-	/**
-	 * @see org.openmrs.module.web.extension.AdministrationSectionExt#getLinks()
-	 */
-	public Map<String, String> getLinks() {
-		
-		Map<String, String> map = new HashMap<String, String>();
-		
-		map.put("module/ohricore/ohricore.form", "ohricore.title");
-		
-		return map;
-	}
-	
-}
+//public class AdminList extends AdministrationSectionExt {
```

Comment:
@smallgod I requested you to delete this file

Reply:
My bad. Missed this, had only focused on the API module. Fixed!